app/js/views/services/camera.jsx

```diff
@@ -15,28 +15,60 @@ export default class CameraService extends React.Component {
     };
   }
 
+  /**
+   * Gets service operation with the specified alias.
```

Comment:
nit: I usually add a line break between the description and the JSDoc annotations. Then all the annotation are grouped together. What do you think?